### ISSUE TITLE
The syntax highlighting of all source code files not work on the newest v3.7.2 on the windows 10 20H2

### ISSUE THREAD

**Issue by supersu097** [QL-Win/QuickLook#1210]
*For all's convenience, please use English to fill this issue.*

*Please search within existing issues before the submission. Has this issue already been reported?*
nope
**Describe the bug**
The syntax highlighting of all source code files disappear 

**To Reproduce**
as above

**Expected behavior**
syntax highlighting should work as before

**Desktop (please complete the following information):**
 - OS Version:  windows 10 20H2
 - QuickLook Version: v3.7.2
 - Microsoft Edge WebView2 Runtime 107.0.1418.35

**Screenshots and Exception Log**
If applicable, add screenshots and logs to help explain your problem. The log file is located in [one of several locations](https://github.com/QL-Win/QuickLook/wiki/Differences-Between-Distributions#user-data-location).

**Comment by rabelux** [comment]
Are you using dark mode?

**Comment by supersu097** [comment]
> Are you using dark mode?

nope, I'm not using the dark mode, now i rollback to v3.7.1 and everything works fine, but i really really hope to use the newest version, so sad.

**Comment by mooflu** [comment]
Looks like the syntax files aren't included in the build anymore. 
They were moved to a subfolder but are now tagged to not be copied:

  <Content Include="Syntax\**">
      <CopyToOutputDirectory>None</CopyToOutputDirectory>
  </Content>

https://github.com/QL-Win/QuickLook/commit/660dc88001868af4b00dd1791fa03a7bf0f6d03f#diff-fd4260eb87ae71145d63d99b37eb3b5433e1b6385f6c114f42a8b30f1cf61186R92

When I add them back, it fails to load the "HTML.xshd" syntax file which references the javascript syntax definition which hasn't loaded yet. Probably best to just remove the HTML syntax file, since html viewer plugin would be used for html files anyway.

**Comment by xupefei** [comment]
Hmm that is strange. Let me check what's happening.

**Comment by supersu097** [comment]
Hi all, how is it going here? I'm looking forward to seeing the good news.

**Comment by Solitario119** [comment]
I have the same problem too.
